crates/control/Cargo.toml

```diff
@@ -20,6 +21,7 @@ projection = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 serialize_hierarchy = { workspace = true }
+simple-websockets = { workspace = true }
```

Comment:
Are these two dependencies added here used at all?
I don't see them being used anywhere.

Reply:
removed

Reply:
There's still the byteorder dependency.
I don't think that's used either.

Reply:
removed byteorder dependency from control crate